scrape.js

```diff
@@ -57,6 +61,11 @@ const autoScroll = async (page) => {
   });
 };
 
+/**
+ * Fetch all profile links
+ * @param {Promise} page Promise of Browser page
+ * @param {Number} pagesToVisit Static param, setted to 2
```

Comment:
The `pagesToVisit` argument can use a more informative description. For example, `Number of pages to scrape(defaults to 2)`